dynamodb/src/main/scala/uk/gov/nationalarchives/DADynamoDBClient.scala

```diff
@@ -76,7 +75,7 @@ trait DADynamoDBClient[F[_]: Async]:
   /** @param tableName
     *   The name of the table
     * @param gsiName
```

Comment:
This name doesn't match the parameter name

Reply:
Ah yeah, good spot, I've changed it.